atomic_reactor/util.py

```diff
@@ -916,6 +916,41 @@ def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
     return ManifestDigest(**digests)
 
 
+def get_manifest_list(image, registry, insecure=False, dockercfg_path=None):
```

Comment:
Any possibility this could take a `version` parameter and be used by `get_manifest_digests()`, to reduce duplication?

Reply:
mmm and how would I use it by get_manifest_digest() ? as that is collection versions: Docker-Content-Digest dict? and for manifest list we don't have it?

Reply:
made general get_manifest function, to reduce duplication